packages/iframe-stamper/src/index.ts

```diff
@@ -233,35 +265,14 @@ export class IframeStamper {
     organizationId: string,
     keyFormat?: KeyFormat
   ): Promise<boolean> {
```

Comment:
Not sure why TS isn't complaining about this, but I believe there's a type mismatch? The return type is `Promise<boolean>`, but `addMessageHandler` returns `Promise<any>`.